pegasus/demo/demo_workflow.py

```diff
@@ -0,0 +1,69 @@
+import logging
+import sys
+from pathlib import Path
+from Pegasus.api import *
+
+logging.basicConfig(level=logging.DEBUG, stream=sys.stdout)
+home = Path.home().absolute()
+shared_scratch_dir = f"{home}/workflows/scratch"
+local_storage_dir = f"{home}/workflows/output"
+sc = SiteCatalog()
+sc.add_sites(
+    Site("local", arch=Arch.X86_64, os_type=OS.LINUX).add_directories(
+        Directory(Directory.SHARED_SCRATCH, shared_scratch_dir).add_file_servers(
+            FileServer("file://" + shared_scratch_dir, Operation.ALL)
+        ),
+        Directory(Directory.LOCAL_STORAGE, local_storage_dir).add_file_servers(
+            FileServer("file://" + local_storage_dir, Operation.ALL)
+        ),
+    )
+)
+
+saga = Site("saga", arch=Arch.X86_64, os_type=OS.LINUX)
+# TODO? partition: scavenge
+
+saga.add_directories(
+    Directory(Directory.SHARED_SCRATCH, shared_scratch_dir).add_file_servers(
+        FileServer("file://" + shared_scratch_dir, Operation.ALL)
+    )
+)
+
+saga.add_env(
+    key="PEGASUS_HOME", value="/nas/gaia/shared/cluster/pegasus5/pegasus-5.0.0"
+)
+
+saga.add_pegasus_profile(style="glite", auxillary_local=True)
```

Comment:
```suggestion
saga.add_pegasus_profile(style="glite", auxillary_local=True, data_configuration="sharedfs")
```
@danielnapierski -- Can you see if adding this makes a difference on your workflow? The default `data_configuration` changed in Pegasus 5.0 from `sharedfs` to `condorio`. This may be relevant because the failure appears to be something related to data transfers?
> [2] java.lang.RuntimeException: Unable to modify job py36_version_ID0000001 for worker node execution by SLS backend using Condor File Transfers to the worker node at edu.isi.pegasus.planner.code.gridstart.PegasusLite.wrapJobWithPegasusLite(PegasusLite.java:1098)

Reply:
Thanks Jacob this change was needed and I've added it.  Next I have to specify the partition to use.  https://github.com/GAIA-AIDA/aida-integration/pull/22/files#diff-f867d610acddad8f8ea58050228717c6f8c62187b6768d215b9d8026ffe75f8dR67